lib/css/modules/_showImages.scss

```diff
@@ -317,6 +319,23 @@ div.madeVisible {
 			padding: 0 1%;
 			box-sizing: border-box;
 
+			.video-advanced-position-cursor {
+				display: none;
+				position: relative;
+				top: -9px;
+				height: 16px;
+				width: 16px;
+				margin-left: calc(-1% + -9px);
```

Comment:
The width of `.video-advanced-progress` was 98% (by `padding: 0 1%`), necessitating `calc`. Updated now by removing `padding`.